tests/unittest.py

```diff
@@ -169,15 +172,13 @@ def get_builder(self, network: str) -> TestBuilder:
             .set_network(network)
         return builder
 
-    def create_peer_from_builder(self, builder, start_manager=True):
+    def create_peer_from_builder(self, builder: Builder, start_manager: bool = True) -> HathorManager:
         artifacts = builder.build()
         manager = artifacts.manager
 
         if artifacts.rocksdb_storage:
             self._pending_cleanups.append(artifacts.rocksdb_storage.close)
 
-        manager.avg_time_between_blocks = 0.0001
```

Comment:
Why are you changing this?

Reply:
After this test was typed, mypy complained that `HathorManager` does not have an `avg_time_between_blocks` property. Therefore, this was useless (probably an artifact from some older code) and removing it doesn't break the test.

Reply:
Readded as a commented line in https://github.com/HathorNetwork/hathor-core/pull/974/commits/6ff78af167152e8f0737a5f46f6096ccc0a6c419, as discussed in https://github.com/HathorNetwork/hathor-core/pull/973#discussion_r1534692226

Reply:
Thanks!